index.bs

```diff
@@ -3190,6 +3190,22 @@ value).</p>
 
    </dd>
 
+   <dt><dfn>HTML character reference in annotation state</dfn></dt>
+
+   <dd>
+
+    <p>Attempt to <a>consume an HTML character reference</a>, with the <a spec=html>additional
```

Comment:
I think this is right, because HTML says "optionally with an additional allowed character, which, if specified where the algorithm is invoked, adds a character to the list of characters that cause there to not be a character reference."

Without any context, I assumed this meant that `>` would be allowed as part of the character reference, which would the opposite of the intended.